tools/cli/internal/openapi/oasdiff.go

```diff
@@ -76,7 +76,7 @@ func (o OasDiff) mergePaths() error {
 		if originalPathData := basePaths.Value(path); originalPathData == nil {
 			basePaths.Set(path, removeExternalRefs(externalPathData))
 		} else {
-			if err := o.handlePathConflict(originalPathData, path); err != nil {
+			if err := o.handlePathConflict(externalPathData, path); err != nil {
```

Comment:
main change: we were getting the originalPathData (baseData, which is mms, but we expect annotations to be in SOA)

Reply:
What does it mean "_expect annotations to be in SOA_"? Do you mean the file path of the spec was MMS but we need the service spec instead?

Reply:
expect annotations to be in Standalone service, correct. We were adding the annotation to MMS, but it should live in the standalone service once the copy endpoint is created